src/main/java/com/google/gcp_variant_transforms/library/SchemaGeneratorImpl.java

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import com.google.common.collect.ImmutableList;
+import htsjdk.variant.vcf.VCFHeader;
+
+
+
+/**
+ * Service to create BigQuery Schema from VCFHeader 
+ */
+public class SchemaGeneratorImpl implements SchemaGenerator {
+
+  public static class FieldDescriptionConstants {
+    public static final String REFERENCE_NAME = "Reference name.";
+    public static final String START_POSITION = "Start position. Corresponds to the first " +
```

Comment:
`"Start position (1-based). ..."`
Please make sure you exactly copy [all the descriptions](https://github.com/googlegenomics/gcp-variant-transforms/blob/ee3b767c26ddc513895fe9285ab952ce2a52dcfa/gcp_variant_transforms/libs/schema_converter.py#L91).

Reply:
Got it! This is the only description that is different. It was mentioned that in our java implementation it's always 1-based (never 0-based), so I wasn't sure about this piece. I will fix it!